### PR TITLE
as: Rename config to `packages`

### PR DESCRIPTION
**Pull request by neoaggelos**
#### Summary

References https://github.com/TheThingsIndustries/lorawan-stack/pull/2295.

This is not breaking config, there are no config options for application packages in TTS.

### CODE REVIEW

**Review by adriansmares — COMMENTED** [review]
`ApplicationPackagesConfig` contains the registry configuration, so I don't see why this is backwards compatible with already existing configurations.

**Comment by neoaggelos** [comment]
> `ApplicationPackagesConfig` contains the registry configuration, so I don't see why this is backwards compatible with already existing configurations.

That is not user configuration though, this is configured in https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/cmd/ttn-lw-stack/commands/start.go#L244-L246

File: pkg/applicationserver/config.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ type Config struct {
 	MQTT                config.MQTT               `name:"mqtt" description:"MQTT configuration"`
 	Webhooks            WebhooksConfig            `name:"webhooks" description:"Webhooks configuration"`
 	PubSub              PubSubConfig              `name:"pubsub" description:"Pub/sub messaging configuration"`
-	ApplicationPackages ApplicationPackagesConfig `name:"application-packages" description:"Application packages configuration"`
+	ApplicationPackages ApplicationPackagesConfig `name:"packages" description:"Application packages configuration"`
```
Comment (adriansmares) [inline pkg/applicationserver/config.go]: `Packages ApplicationPackagesConfig`

**Review by adriansmares — APPROVED** [review]
LGTM but cc @johanstokking

**Review by johanstokking — APPROVED** [review]

File: pkg/applicationserver/config.go (hunk 1 above)
Comment (johanstokking) [reply pkg/applicationserver/config.go]: Isn't it then `name:"-"` ?

Comment (adriansmares) [reply pkg/applicationserver/config.go]: We need the actual name in proprietary.

**Pull request merged**